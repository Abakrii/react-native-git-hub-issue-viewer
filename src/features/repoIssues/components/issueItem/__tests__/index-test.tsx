import React from 'react';
import renderer from 'react-test-renderer';
import IssueItem from '../';

const items = {
  id: 112,
  title: 'render title',
  number: 1,
  body: 'render body',
};
test('renders correctly', () => {
  const tree = renderer.create(<IssueItem {...items} />).toJSON();
  expect(tree).toMatchSnapshot();
});
