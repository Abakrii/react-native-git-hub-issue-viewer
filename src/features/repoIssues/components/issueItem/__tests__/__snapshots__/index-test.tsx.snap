// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "borderColor": "#BBB",
      "borderRadius": 10,
      "borderStyle": "dashed",
      "borderWidth": 1,
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "space-between",
      "marginTop": 16,
      "padding": 16,
    }
  }
>
  <Text
    style={
      Object {
        "textAlign": "center",
        "width": "70%",
      }
    }
  >
    number: 
    1
  </Text>
  <Text
    style={
      Object {
        "textAlign": "center",
        "width": "70%",
      }
    }
  >
    id: 
    112
  </Text>
  <Text
    style={
      Object {
        "textAlign": "center",
        "width": "70%",
      }
    }
  >
    title: 
    render title
  </Text>
  <Text
    style={
      Object {
        "textAlign": "center",
        "width": "70%",
      }
    }
  >
    body: 
    render body
  </Text>
</View>
`;
